### PR TITLE
Add secret for private git

### PR DESCRIPTION
**Pull request by keigohtr**
## What is this PR for?

Add Git SSH Key management function for accessing a private git repository.

## This PR includes

- Use `Kubernetes Secret` to store Git SSH Key
- Add WebUI for Git SSH Key
- Add test

## What type of PR is it?

Feature

## What is the issue?

https://github.com/rekcurd/dashboard/issues/76

## How should this be tested?

Run unittest

### CODE REVIEW

**Comment by yoquankara** [comment]
It looks like you don't use encryption. Could you provide rationale for that decision?

**Comment by keigohtr** [comment]
@yoquankara Thank you for your review.
Kubernetes Client does encryption automatically.
```
        v1_secret = client.V1Secret(
            api_version="v1",
            kind="Secret",
            metadata=client.V1ObjectMeta(
                name=name,
                namespace=service_level,
                labels={"rekcurd-worker": "True", "id": application_model.application_id,
                        "name": application_model.application_name}
            ),
            string_data=string_data,
            type="Opaque")
```

And when we use them, we need to decode it -> https://github.com/rekcurd/dashboard/pull/94/files#diff-6e60f8497f16de75099b3715931c19c8R944

**Comment by yoquankara** [comment]
Base64 is simple to decode, so I wondered if it was better to add extra encryption as suggested here.

> https://kubernetes.io/docs/concepts/configuration/secret/#risks
> https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/

That said, encrypt data at rest is beta feature in v1.13 though. So we may consider it later.

**Comment by keigohtr** [comment]
@yoquankara Yes I think so too.
This is current Kubernetes spec, so they will improve `secret` in the future. I know this situation and I expect to use a better encryption algorithm like bcrypt in the future.

**Comment by keigohtr** [comment]
@yoquankara Thank you veeeeery much!! :)

File: rekcurd_dashboard/apis/api_kubernetes_secret.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    @kubernetes_secret_api_namespace.marshal_with(success_or_not)
+    @kubernetes_secret_api_namespace.expect(git_key_parser)
+    def post(self, project_id: int, application_id: str):
+        """Save git key."""
+        args = self.git_key_parser.parse_args()
+        service_level = args["service_level"]
+        try:
+            string_data = load_secret(project_id, application_id, service_level, GIT_SECRET_PREFIX)
+        except:
```
Comment (yuki-mt) [inline rekcurd_dashboard/apis/api_kubernetes_secret.py]: I think we should display message when no kubernetes is registered,
so what about the following?

```
try:
   ...
except NotFound:
    raise RekcurdDashboardException("No Kubernetes is registered")
except:
    straing_data = dict()
...
```

Comment (keigohtr) [reply rekcurd_dashboard/apis/api_kubernetes_secret.py]: Great suggestion!
Cloud you give this a PR?

Comment (yuki-mt) [reply rekcurd_dashboard/apis/api_kubernetes_secret.py]: @keigohtr 
OK! I did. https://github.com/rekcurd/dashboard/pull/102

**Pull request merged**